assets/scripts/main.js

```diff
@@ -127,4 +127,49 @@
         });
     }
   });
+
+  // Add ajax add to cart
+  $(document).ready(() => {
+    $('.single_add_to_cart_button').click((e) => {
+      e.preventDefault();
+      const $thisbutton = $(e.target);
+      const $form = $thisbutton.closest('form.cart');
+      const id = $thisbutton.val();
+      const productQty = $form.find('input[name=quantity]').val() || 1;
+      const productId = $form.find('input[name=product_id]').val() || id;
+      const variationId = $form.find('input[name=variation_id]').val() || 0;
+      const data = {
+        action: 'woocommerceAjaxAddToCart',
+        product_id: productId,
+        product_sku: '',
+        quantity: productQty,
+        variation_id: variationId
+      };
+      $.ajax({
+        type: 'post',
+        url: shop_standards_settings.ajax_url,
+        data,
+        beforeSend() {
+          $thisbutton.removeClass('added').addClass('loading');
+        },
+        complete() {
+          $thisbutton.addClass('added').removeClass('loading');
+        },
+        success(response) {
+          if (response.error && response.product_url) {
+            window.location = response.product_url;
+          } else {
```

Comment:
```suggestion
          }
          else {
```
Again, it looks like the code standards is not highlighting format errors for you.